helm/happa/values.yaml

```diff
@@ -57,7 +59,6 @@ kubernetes:
 happa:
   address: ""
   host: ""
-  letsencrypt: false
```

Comment:
can't find a reason why we would want a different settings on happa and happaapi so moving all under ingress block